examples/.env

```diff
@@ -0,0 +1,7 @@
+MARIADB_ROOT_PASSWORD=secret
+MYSQL_INITDB_SKIP_TZINFO=Y
+MARIADB_REPLICATION_USER=repl
+MARIADB_REPLICATION_PASSWORD=replicationpass
+PRIMARY_name='mariadb-primary'
+REPLICATION_name_1='mariadb-replica-1'
+REPLICATION_name_2='mariadb-replica-2'
```

Comment:
The `REPLICATION_*` are only used once. Can we just omit them and get the default configuration.